src/storage/__tests__/migrations-test.js

```diff
@@ -106,13 +107,33 @@ describe('migrations', () => {
   const base62 = {
     ...base52,
     migrations: { version: 62 },
-    accounts: base52.accounts.map(a => ({ ...a, silenceServerPushSetupWarnings: false })),
+    accounts: (base52.accounts.map(a => ({
+      ...a,
+      silenceServerPushSetupWarnings: false,
+    })): $ReadOnlyArray<{|
+      +email: string,
+      +apiKey: string,
+      +realm: URL,
+      +ackedPushToken: string | null,
+      +zulipFeatureLevel: number | null,
+      +zulipVersion: ZulipVersion | null,
+      +lastDismissedServerPushSetupNotice: Date | null,
+      +userId: UserId | null,
+      +silenceServerPushSetupWarnings: boolean,
+    |}>),
```

Comment:
```suggestion
    // $FlowIgnore[prop-missing] same type-lie as in the test, below at end
    accounts: base52.accounts.map(a => ({
      ...a,
      silenceServerPushSetupWarnings: false,
    })),
```

Weird that for this one piece of data, Flow puts the error up here; for the rest, the error is down at the bottom where we pretend that these old states meet the current type, and so we have `$FlowIgnore` comments there accordingly:
```js
      // $FlowIgnore[incompatible-exact]
      // $FlowIgnore[incompatible-type]
      /* $FlowIgnore[prop-missing]
         this really is a lie -- and kind of central to migration */
      const incomingState: $Rest<GlobalState, { ... }> = before;
```

But the same discrepancy applies to basically all of these except the one or two at the end. So best to keep succinct how we tell Flow to avert its eyes.